cli/src/replace_node/deploy-replacement

```diff
@@ -399,16 +399,20 @@ ping -c1 ${repl_node_hostname} >/dev/null 2>&1 \
 # Update salt states
 update_salt_n_system
 
+l_info "=================================================="
+l_info "Stage 1.1: Applying update repo (if exists)..."
+l_info "=================================================="
+$cmd salt "*" state.apply components.misc_pkgs.swupdate.repo ${salt_opts}
```

Comment:
I realized that this might be applied only for new node. But it's an optimization. Up to you.